packages/Modal/src/Modal.js

```diff
@@ -49,13 +54,8 @@ const Modal = props => {
     >
       {state => (
         <sc.Modal size={size} data-pka-anchor="modal.wrapper" {...moreProps}>
-          <sc.Dialog state={state} role="dialog" aria-modal="true" aria-label={ariaLabel} data-pka-anchor="modal">
-            {headerExtracted && <sc.Header {...headerExtracted.props} onClose={onClose} />}
-            <sc.ContentWrapper role="region" tabIndex="0">
-              {contentExtracted && <sc.Content {...contentExtracted.props} />}
-              {children}
-            </sc.ContentWrapper>
-            {footerExtracted && <sc.Footer {...footerExtracted.props} />}
+          <sc.Dialog state={state} role="dialog" aria-modal aria-label={ariaLabel} data-pka-anchor="modal">
+            <ModalContext.Provider value={contextValue}>{children}</ModalContext.Provider>
```

Comment:
super `nip:` could you move the const `contextValue` closer to the return statement? so is easier to spot the values that will be share across the context 🙏